contributors/design-proposals/storage/raw-block-ro.md

```diff
@@ -0,0 +1,139 @@
+# Raw Block Read-Only Consumption in Kubernetes
```

Comment:
It might be better to add this to the existing block proposal instead of creating a new one

Reply:
@msau42 it was already merged..though you said to create a new one?

Reply:
@msau42 @erinboyd I think you can update merged document to branch.

Reply:
Create a new PR, but add to the existing doc.